Lib/zipfile/__init__.py

```diff
@@ -1585,7 +1585,8 @@ def comment(self, comment):
         self._didModify = True
 
     def read(self, name, pwd=None):
-        """Return file bytes for name."""
+        """Return file bytes for name. 'pwd' is the password to decrypt
+        encrypted files."""
```

Comment:
[The docs](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.read) say:

> *pwd* is the password used for encrypted files as a [bytes](https://docs.python.org/3/library/stdtypes.html#bytes) object and, if specified, overrides the default password set with [setpassword()](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.setpassword)

Reply:
The docstrings in this module leave a lot to be desired overall. The existing proposed text is at least consistent with other descriptions of `pwd=` that already exist.  such as on open() "pwd is the password to decrypt files."

Reply:
Fine with me. I'd consider this, though:

```suggestion
        """Return file bytes for name.
        
        'pwd' is the password to decrypt encrypted files."""
```

Reply:
I think the automerge happened without this, feel free to make it a followup PR if desired.